src/platform/web.rs

```diff
@@ -82,7 +82,7 @@ pub trait WindowExtWebSys {
     /// would prevent that.
     ///
     /// Some events are impossible to prevent. E.g. Firefox allows to access the native browser
-    /// context menu with Shift+Rightclick.
+    /// context menu with Shift+Right-click.
```

Comment:
looks arbitrary.

Reply:
Generally for combination words (like Rightclick) I try to split it into two words unless it isn't possible (it's part of public API).